bBridgeAPISDK.Test/TestAuthorization.cs

```diff
@@ -11,15 +12,18 @@ public class TestAuthorization
         [Fact]
         public void TestCanReceiveAuthorizationTokenForUserNameAndPassword()
         {
-            var baseUri = new Uri(Resources.bBridgeAPIBaseURI);
-
             IAuthorizer userPasswordAuthorizer = new LazyCredentialsAuthorizer(
-                Resources.bBridgeAPIUserName,
-                Resources.bBridgeAPIPassword,
-                baseUri);
+				string.IsNullOrEmpty(TestResources.bBridgeAPIUserName) ?
+					Environment.GetEnvironmentVariable("BBRIDGE_API_USER_NAME") : 
+					TestResources.bBridgeAPIUserName,
+				string.IsNullOrEmpty(TestResources.bBridgeAPIUserName) ?
+					Environment.GetEnvironmentVariable("BBRIDGE_API_PASSWORD") :
+					TestResources.bBridgeAPIPassword,
+                Path.Combine(TestResources.bBridgeAPIBaseURI,
+                TestResources.bBridgeAPIAuthUrlSuffix));
```

Comment:
You use same code to create `LazyCredentialsAuthorizer` at least 4 times. Maybe move it to some common method?